scripts/psij-consol.py

```diff
@@ -0,0 +1,95 @@
+#! /usr/bin/python3
+
+import psij
+from psij import JobExecutor
+from psij import JobSpec
+import sys
+import os
+import argparse
+from psij import Import
+
+
+
+
+parser = argparse.ArgumentParser(prog='psij-consol')
+subparser = parser.add_subparsers(dest="command", help='Subcommands')
+validate_parser = subparser.add_parser("validate", help='validate JobSpec file')
+validate_parser.add_argument("file", help="JobSpec file")
+execute_parser = subparser.add_parser("run", help='execute JobSpec file')
+execute_parser.add_argument("file", help="JobSpec file")
+execute_parser.add_argument("-j", 
+                            "--job-executor",
+                            dest = "executor",
+                            choices = ["cobalt"
+                                       , "local"
+                                       , "batch-test"
+                                       , "flux"
```

Comment:
The comma should be after the entry, and not before the next entry? But if you like it this way, that's OK I guess. (I really don't like it though.)

Reply:
Changed it